io.gitlab.sulincix.sterm.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: io.gitlab.sulincix.sulincix
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=LD_LIBRARY_PATH=/app/lib64:/app/lib
+  - --share=ipc
+command: sterm
+modules:
+  - name: vte
+    buildsystem: meson
+    config-opts:
+      - -Dgtk4=false
+      - -Dgtk3=true
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/vte/-/archive/0.78.2/vte-0.78.2.tar.bz2
+        sha256: c655778c2148c30f7416a450d7d9dad7bf481d3411896095aacd92a448c3ebf3
+
+  - name: host-spawn
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 host-spawn /app/bin/host-spawn
+    sources:
+      - type: file
+        url: https://github.com/1player/host-spawn/releases/download/v1.6.0/host-spawn-x86_64
+        sha256: 75159f0d3924a0bd6d230c6414d19ad44609b5a16671db6624681c910f9bb4ff
+        dest-filename: host-spawn
+        only-arches: [x86_64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/1player/host-spawn/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test(".+x86_64$"))][0] | .browser_download_url'
+      - type: file
+        url: https://github.com/1player/host-spawn/releases/download/v1.6.0/host-spawn-aarch64
+        sha256: 00f9617c139b9447e6c453a27602ca87f28cdac3aa62b04dcb38d215b2758165
+        dest-filename: host-spawn
+        only-arches: [aarch64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/1player/host-spawn/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test(".+aarch64$"))][0] | .browser_download_url'
+
+  - name: sterm
+    buildsystem: meson
+    build-options:
+      env:
+        PKG_CONFIG_PATH: /app/lib64/pkgconfig:/app/lib/pkgconfig
```

Comment:
this is unecessary.

also changing the pkg-config path should always be done with `append-pkg-config-path` or the `prepend-` equivalent.

Reply:
see also https://github.com/flathub/flathub/pull/5898#discussion_r1864898074